test/test_jit.py

```diff
@@ -11922,6 +11922,8 @@ def forward(self, x):
         weak_mod.weight = torch.nn.Parameter(torch.ones(5, 5) * 100)
         self.assertFalse(strong_mod(inp).allclose(weak_mod(inp)))
 
+    @unittest.skipIf(hasattr(torch.jit, 'WeakScriptModuleProxy'), "# TODO: re-enable"
```

Comment:
re-enable this when what lol

Reply:
The full message is split over 2 strings